src/hub.js

```diff
@@ -303,6 +303,9 @@ async function updateEnvironmentForHub(hub) {
   if (hub.scene) {
     isLegacyBundle = false;
     sceneUrl = hub.scene.model_url;
+  } else if (hub.scene === null) {
```

Comment:
Does this actually work for existing hubs that point to a scene that is now removed/delisted? AFAICT, `join_with_hub` will still preload the scene data associated with the hub.

Reply:
Ah, hub_channel jsonification goes through scene_view